docs/plugins.md

```diff
@@ -38,6 +38,12 @@ If the plugin is unable to be found automatically, you can load them with:
 * [`@prettier/plugin-php`](https://github.com/prettier/prettier-php)
 * [`@prettier/plugin-swift`](https://github.com/prettier/prettier-swift)
 
+## Unofficial Plugins
+
+* [prettier-plugin-java](https://github.com/thorbenvh8/prettier-java)
+* [@iamsolankiamit/prettier-ruby](https://github.com/iamsolankiamit/prettier-ruby)
```

Comment:
The `@`s are still incorrect as it implies it is an npm package name. Either `iamsolankiamit/prettier-ruby` or `prettier-plugin-ruby` is fine.

Reply:
Really confused.. :| Is this fine now?